Bottom: 0a6efd67db4c36728192f7c77485b78ff81f6caf
Top:    7e869edce7c679fc9f459d72ae62a1a64ccd26bb
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-07-24 16:06:23 +0100

Refresh of use-trackball-to-allow-rotatio

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 1b9fbce..21d0b84 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -1019,8 +1019,8 @@ ghid_drawing_area_expose_cb (GtkWidget *widget,
   min_x = MIN (min_x, new_x);  max_x = MAX (max_x, new_x);
   min_y = MIN (min_y, new_y);  max_y = MAX (max_y, new_y);
 
-  region.X1 = min_x; region.X2 = max_x + 1;
-  region.Y1 = min_y; region.Y2 = max_y + 1;
+  region.X1 = min_x;  region.X2 = max_x + 1;
+  region.Y1 = min_y;  region.Y2 = max_y + 1;
 
   glColor3f (port->bg_color.red / 65535.,
              port->bg_color.green / 65535.,
